ui/src/app/edge/live/autarchy/autarchy.component.html

```diff
@@ -0,0 +1,22 @@
+<ng-container *ngIf="service.currentEdge | async as edge">
+    <ng-container *ngIf="edge.currentData | async as currentData">
+        <ion-card>
+            <ion-item lines="full" color="light">
+                <ion-avatar slot="start">
+                    <img src="assets/img/autarchy.png">
+                </ion-avatar>
+                <ion-label translate>General.Autarchy</ion-label>
+            </ion-item>
+            <ion-card-content *ngIf="currentData.summary as sum">
+                <ion-row>
+                    <svg width="100%" height="20">
+                        <rect fill-opacity="1" width="100%" height="20" style="fill:#f4f4f4" />
+                        <rect x="1" y="2" attr.width="{{sum.system.autarchy}}%" height="16" style="fill:#2d8fab" />
+                        <text x="50%" y="58%" dominant-baseline="middle"
+                            text-anchor="middle">{{sum.system.autarchy | number:'1.0-0'}}%</text>
+                    </svg>
```

Comment:
Should be a separate component that can be included.

<percentage-bar [value]="sum.system.autarchy"></percentage-bar>